src/app/zones/views/ZoneDetailView.vue

```diff
@@ -102,14 +106,79 @@
             v-if="props.data.zoneInsight.subscriptions.length > 0"
           >
             <h2>{{ t('zone-cps.detail.subscriptions') }}</h2>
-
-            <KCard class="mt-4">
-              <SubscriptionList
-                :subscriptions="props.data.zoneInsight.subscriptions"
+            <AppCollection
+              :headers="[
+                { ...me.get('headers.name'), label: 'Name', key: 'name' },
```

Comment:
Gotcha, will sort 👍

Reply:
I just noticed while doing this: A few days back I think we decided to change this piece of text to `Zone Leader instance ID`. So I'm 99% sure this column should be the same.

I'll update the screengrab below to use `Zone Leader instance ID` in the column also (instead of just `Zone Instance ID`)

Shout if that ^ isn't correct, otherwise I'll resolve this once I'm done.

![Screenshot 2024-07-30 at 16 29 32](https://github.com/user-attachments/assets/a6ef9a2b-46bb-4ea9-81e5-47d560afe8dd)